workflow/controller/operator.go

```diff
@@ -116,8 +116,21 @@ var (
 
 // maxOperationTime is the maximum time a workflow operation is allowed to run
 // for before requeuing the workflow onto the workqueue.
-const maxOperationTime = 10 * time.Second
-const defaultRequeueTime = maxOperationTime
+var maxOperationTime = 10 * time.Second
```

Comment:
@jessesuen @sarabala1979 this change allows you to configure the `maxOperationTime`. I've confirm this with a user that higher values (30s tested) prevent zombie worklows. We increase the number of workflow workers from 8 to 32 back in May. I think we should set this to 20s by default. Thoughts?